src/main/core-impl/java/com/mysql/cj/log/BaseMetricsHolder.java

```diff
@@ -372,4 +372,27 @@ public boolean checkAbonormallyLongQuery(long millisOrNanos) {
         return res;
 
     }
+
+    public void resetMetrics() {
+        longestQueryTimeMs = 0;
+        maximumNumberTablesAccessed = 0;
+        minimumNumberTablesAccessed = Long.MAX_VALUE;
```

Comment:
That's a strange default value. Should it be 0?

Reply:
These were the original values that are set when the object is created.